starknet_py/net/schemas/rpc.py

```diff
@@ -297,6 +331,26 @@ def make_dataclass(self, data, **kwargs) -> BlockStateUpdate:
         )
 
 
+class PendingBlockStateUpdateSchema(Schema):
+    old_root = Felt(data_key="old_root", required=True)
+    state_diff = fields.Nested(StateDiffSchema(), data_key="state_diff", required=True)
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> PendingBlockStateUpdate:
+        return PendingBlockStateUpdate(
+            **data,
+        )
```

Comment:
Could this be one line or formatter wants it like this?

Reply:
It's because of a trailling comma, will remove it.